### ISSUE TITLE
Still can't deny friend requests

### ISSUE THREAD

**Issue by himselfv** [EionRobb/skype4pidgin#628]
I think this worked at some point, but now it says

```
libpurple: skypeweb:auth_reject_cb(): in
libpurple: skypeweb:auth_reject_cb(): buddy_name=xxxxx
libpurple: http:Performing new request 0x1ad4580 to contacts.skype.com.
libpurple: http:Connecting to contacts.skype.com...
libpurple: skypeweb:auth_reject_cb(): out
libpurple: http:locking a (previously used) socket: 0x19e47e0
libpurple: http:Got main header with code 403
libpurple: http:Got headers end
libpurple: http:releasing a socket: 0x19e47e0
libpurple: http:Request 0x1ad4580 performed without success.
```
Using latest code from here. Tried to do the same in Pidgin with the latest release build, same result (can't check the logs but the request reappears on reconnect).

**Comment by dequis** [comment]
Isn't this the same as #577?

**Comment by himselfv** [comment]
Oh... I thought that one had been fixed in [Release 1.5 from May, 2018](https://github.com/EionRobb/skype4pidgin/releases/tag/1.5) and its something new, but looking at the dates, probably not. Thanks!
